config.js

```diff
@@ -1,22 +1,24 @@
 APP.bindKeys("global", {
+    "C-b": APP.compile,
+    "C-r": APP.run,
+    "C-g":APP.compileAndRun,
+    "C-f": APP.findFile,
     "C-o":APP.focusFilter,
     "C-c g":APP.displayGeneratedCPP,
-    "C-x C-f": APP.findFile,
-    "C-c b": APP.compile,
-    "C-c r": APP.run,
     "F4": APP.stopEmulator,
     "F5": APP.debug,
     "F8": APP.debugContinue,
-    "S-F7": APP.debugStepIn,
-    "F7": APP.debugStepOver,
+    "F10": APP.debugStepOver,
+    "F11": APP.debugStepIn,
     "F6": APP.debugStepOut,
-    "C-x C-c": APP.exit,
-    "C-c d": _=>nw.Window.get().showDevTools(),
+    "C-q C-c": APP.exit,
```

Comment:
`C-q C-c` is a bit of an odd shortcut... how about just `C-q` instead?
devtools could be `C-S-j` (same as chrome)